dlt/common/libs/pydantic.py

```diff
@@ -121,14 +128,32 @@ def pydantic_to_table_schema_columns(
             # try to coerce unknown type to text
             data_type = "text"
```

Comment:
please detect pydantic model in exception handler. if detected: data_type is complex and set the flag is_inner_type_pydantic_model

Reply:
then we set `data_type=complex` if it is pydantic model right?